bigframes/session/__init__.py

```diff
@@ -1373,7 +1373,14 @@ def _start_query_ml_ddl(
         # https://cloud.google.com/bigquery/docs/customer-managed-encryption#encrypt-model
         job_config.destination_encryption_configuration = None
 
-        return bf_io_bigquery.start_query_with_client(self.bqclient, sql, job_config)
+        results_iterator, query_job = bf_io_bigquery.start_query_with_client(
```

Comment:
I see `start_query_with_client` already has the stats update

https://github.com/googleapis/python-bigquery-dataframes/blob/fd06d31f5e7edab3ce3af9a4e3e4415a4463ee5d/bigframes/session/_io/bigquery/__init__.py#L245-L246

so feels like we could be adding double counting

Reply:
It's not double counting, but yes here we can send the metric into the start_query_with_client, thanks for point that out.